### ISSUE TITLE
Unicode in commit messages

### ISSUE THREAD

**Issue by nomeata** [sociomantic-tsunami/git-hub#276]
I sometimes use unicode characters in commit messages. If `git hub pull new` tries to use such a commit message as the basis for the PR description, I get
```
~/dfinity/motoko $ git hub pull new 
Traceback (most recent call last):
  File "/home/jojo/.bin/git-hub", line 2205, in <module>
    main()
  File "/home/jojo/.bin/git-hub", line 2197, in main
    args.run(parser, args)
  File "/home/jojo/.bin/git-hub", line 674, in check_config_and_run
    cmd.run(parser, args)
  File "/home/jojo/.bin/git-hub", line 1513, in run
    head_ref, head_name))
  File "/home/jojo/.bin/git-hub", line 1105, in editor
    return editor(cls.help_msg, msg)
  File "/home/jojo/.bin/git-hub", line 326, in editor
    f.write(msg or '')
UnicodeEncodeError: 'ascii' codec can't encode character u'\u221e' in position 4: ordinal not in range(128)
```
This used to work in dccc92b.

A `git bisect` (a wonderful tool) points to 30c0710d2d1752d21856e9847a9831675511fd68

@jean?

**Comment by jean** [comment]
Yeah that looks like me. I'll check it out as soon as I get a chance.

**Comment by nomeata** [comment]
Thanks! ☺

**Comment by jean** [comment]
Looks like it's a Python 2 issue, this worked fine with Python 3:

```
19:55~/git/www2020(master)$ git hub pull new
Pushing master to master in fork
Creating pull request from branch master to pyconth/www2020:master
[10] Test commit message with unicode characters (jean)
     https://github.com/PyConTH/www2020/pull/10
```

**Comment by nomeata** [comment]
Yup – is python 2 no longer supported?
```
~/build/git-hub $ head -n1 git-hub 
#!/usr/bin/env python2
```

**Comment by jean** [comment]
It is meant to be supported. In fact Python 3 only runs after `2to3`.

**Comment by nomeata** [comment]
@jean did you have a chance to look at this?

**Comment by daniel-zullo** [comment]
@nomeata would you mind to test #278? 
@jean you might want to review it

**Comment by llucax** [comment]
This was addressed in #278. Even when there was no confirmation this particular issue was resolved, I'll close because of the lack of response.
